### PR TITLE
various tweaks

### PR DESCRIPTION
**Pull request by geoffthemedio**
pull request to provoke test builds

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,36 +0,0 @@
-os: Visual Studio 2019
```
Comment (geoffthemedio) [inline .appveyor.yml]: @o01eg is there any reason we need the AppVeyor builds still?

Comment (o01eg) [reply .appveyor.yml]: Not sure, maybe we left it as alternative to Github Actions

**Comment by geoffthemedio** [comment]
@Cjkjvfnby @Grummel7 any issues with the Python adjustments in this?

**Comment by Cjkjvfnby** [comment]
Python skeletons could be updated later. As I understand, nothing is changed to AI in interface.

**Pull request merged**